.gitignore

```diff
@@ -6,6 +6,7 @@ venv
 my-db*
 .history
 arrow-datafusion/
+arrow-datafusion-python
```

Comment:
digging through the script it appears it uses the python bindings, so I am starting to add instructions on how to build/install them